resources/lang/en/general.php

```diff
@@ -87,7 +87,10 @@
     'firstname_lastname_format'	=> 'First Name Last Name (jane.smith@example.com)',
     'firstname_lastname_underscore_format' => 'First Name Last Name (jane_smith@example.com)',
     'lastnamefirstinitial_format' =>  'Last Name First Initial (smithj@example.com)',
-    'first'					=> 'First',
+     'first'					=> 'First',
+    'firstnamelastname'    => 'First Name Last Name (janesmith_@example.com)',
+    'lastname_firstinitial' => 'Last Name First Initial (smith_f@example.com)',
+    'firstinitial.lastname' => 'First Initial Last Name (jane.smith@example.com)',
```

Comment:
This says "firstinitial.lastname" but the example email address is firstname.lastname